data/mappings/defaults.hjson

```diff
@@ -64,6 +64,18 @@
             "processor": "STM32F411",
             "bootloader": "stm32-dfu",
             "board": "BONSAI_C4"
+        },
+        "rp2040_ce": {
+            "processor": "RP2040",
+            "bootloader": "rp2040",
+            "board": "QMK_PM2040",
+            "pin_compatible": "rp2040_ce"
+        },
```

Comment:
Thinking about it, my preference would be to not include this here so its not exposed to `new-keyboard`
```suggestion
```